ct.yaml

```diff
@@ -10,6 +10,6 @@ chart-repos:
   - prometheus=https://prometheus-community.github.io/helm-charts
   - open-telemetery=https://open-telemetry.github.io/opentelemetry-helm-charts
 # Some of tests require longer than default timeout
-helm-extra-args: --timeout 15m --debug
+helm-extra-args: --timeout 15m --wait
```

Comment:
This will make our output less verbose and potentially easier to view in github actions.